ui/perfherder/compare/CompareSelectorView.jsx

```diff
@@ -130,7 +130,7 @@ export default class CompareSelectorView extends React.Component {
                 selectedRepo={originalProject}
                 title="Base"
                 checkbox
-                text="By default, Perfherder will compare against performance data gathered over the last 2 days from when new revision was pushed"
+                text="Uncheck and Perfherder will compare against performance data gathered over the last 2 days from when new revision was pushed. Otherwise, you can choose a specific revision as a base."
```

Comment:
I think this may still not be quite right.  Since it is unchecked by default, perhaps the wording should be:

    If checked, you can choose a specific revision as a base.  Otherwise, Perfherder will compare against 
    performance data gathered over the last 2 days from when new revision was pushed.

Reply:
Yeah! You are right! Thanks!